lib/LaTeXML/Package/statements.sty.ltxml

```diff
@@ -159,7 +159,7 @@ sub definitionBody {
       $doc->openElement('dc:title');
       $doc->absorb($title);
       $doc->closeElement('dc:title');}
-    $doc->openElement('omdoc:CMP');
+   if ($doc->isCloseable('omdoc:CMP')) { $doc->openElement('omdoc:CMP'); }
```

Comment:
I do not understand that do we really want to open the element if it is closable?